<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>452</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,20">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Параметры:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>λ:</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLineEdit" name="lineEdit_lm_value">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>μ:</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLineEdit" name="lineEdit_mu_value">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>σ:</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLineEdit" name="lineEdit_sg_value">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Количество заявок:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_count_value">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="textDirection">
             <enum>QProgressBar::TopToBottom</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label_message">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#pushButton {
  border: 1px solid#adadad;
  background-color: rgb(225, 225, 225);
  color: rgb(0, 0, 0);
}

#pushButton:hover {
  border: 1px solid#0078d7;
  background-color: rgb(229, 241, 251);
  color: rgb(0, 0, 0);
}

#pushButton:pressed {  
  border: 1px solid#005499;
  background-color: rgb(204, 228, 247);
  color: rgb(0, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Запуск</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="PlotWidget" name="application_count_graphic" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PlotWidget" name="handler_status_graphic" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>80</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="create_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/create.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/create.png</iconset>
   </property>
   <property name="text">
    <string>Создать...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/open.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/save.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="save_as_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/save_as.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="exit_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/exit.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/file_menu_icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="info_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/info_menu_icons/info.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/info_menu_icons/info.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="resize_btn">
   <property name="icon">
    <iconset>
     <normaloff>../User/Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/resize.png</normaloff>../User/Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/resize.png</iconset>
   </property>
   <property name="text">
    <string>Изменить размер</string>
   </property>
  </action>
  <action name="flip_gorizontally_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/flip_gorizontally.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/flip_gorizontally.png</iconset>
   </property>
   <property name="text">
    <string>Отразить по горизонтали</string>
   </property>
  </action>
  <action name="flip_vertically_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/flip_vertically.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/flip_vertically.png</iconset>
   </property>
   <property name="text">
    <string>Отразить по вертикали</string>
   </property>
  </action>
  <action name="turn_90_right_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/turn_90_right.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/turn_90_right.png</iconset>
   </property>
   <property name="text">
    <string>Повернуть на 90° вправо</string>
   </property>
  </action>
  <action name="turn_90_left_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/turn_90_left.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/turn_90_left.png</iconset>
   </property>
   <property name="text">
    <string>Повернуть на 90° влево</string>
   </property>
  </action>
  <action name="turn_180_btn">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/turn_180.png</normaloff>../Documents/GitHub/PyPaint/icons/menubar_icons/image_menu_icons/turn_180.png</iconset>
   </property>
   <property name="text">
    <string>Повернуть на 180°</string>
   </property>
  </action>
  <action name="increase_btn">
   <property name="icon">
    <iconset>
     <normaloff>../User/Documents/GitHub/PyPaint/icons/menubar_icons/view_menu_icons/increase.png</normaloff>../User/Documents/GitHub/PyPaint/icons/menubar_icons/view_menu_icons/increase.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
  </action>
  <action name="decrease_btn">
   <property name="icon">
    <iconset>
     <normaloff>../User/Documents/GitHub/PyPaint/icons/menubar_icons/view_menu_icons/decrease.png</normaloff>../User/Documents/GitHub/PyPaint/icons/menubar_icons/view_menu_icons/decrease.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
  </action>
  <action name="by_window_size_btn">
   <property name="icon">
    <iconset>
     <normaloff>../User/Documents/GitHub/PyPaint/icons/menubar_icons/view_menu_icons/by_window_size.png</normaloff>../User/Documents/GitHub/PyPaint/icons/menubar_icons/view_menu_icons/by_window_size.png</iconset>
   </property>
   <property name="text">
    <string>По размеру окна</string>
   </property>
  </action>
  <action name="go_next">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/operations_icons/history_icons/next_history_icon.png</normaloff>../Documents/GitHub/PyPaint/icons/operations_icons/history_icons/next_history_icon.png</iconset>
   </property>
   <property name="text">
    <string>Вперед</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="go_back">
   <property name="icon">
    <iconset>
     <normaloff>../Documents/GitHub/PyPaint/icons/operations_icons/history_icons/back_history_icon.png</normaloff>../Documents/GitHub/PyPaint/icons/operations_icons/history_icons/back_history_icon.png</iconset>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
